src/adapter/src/catalog/apply.rs

```diff
@@ -902,7 +951,7 @@ impl CatalogState {
         &mut self,
```

Comment:
Does this need to be `mut`? It looks like it's just `pack` fns. Maybe some parent `mut`s can go away too.

Reply:
Good catch, it does not, just updated this.